pkg/plugin/driver/job/template.go

```diff
@@ -1,8 +1,10 @@
 package job
 
-import "text/template"
+import (
+	"github.com/heptio/sonobuoy/pkg/plugin/driver/utils"
+)
 
-var jobTemplate = template.Must(template.New("jobTemplate").Parse(`
+var jobTemplate = utils.NewTemplate("jobTemplate", `
```

Comment:
@chuckha I thought about that actually, but I thought development would be too much of a pain. Open to suggestions though!

Reply:
what do you mean by development? i think you're much closer to this right now to know why it wouldn't work

Reply:
I mean I think actually creating Daemonsets and Pods is easier to work with in yaml format. Otherwise it's all
```
&v1.Pod{
   Env: []v1.EnvVar{
      {Name: "CA_CERT",
       Value: "SomeValue"}
    },
    Spec: &v1.PodSpec{
       Containers: []v1.Container{
        ...
```

Reply:
It's just really clunky and verbose for a really structure